tests/rt_cascading.erl

```diff
@@ -535,25 +582,66 @@ circle_and_spurs_test_() ->
                 ?assertEqual({error, notfound}, maybe_eventually_exists(N, Bucket, Bin))
             end || N <- Nodes, N =/= NorthSpur]
         end},
+
         {"check pendings", fun() ->
             wait_until_pending_count_zero(Nodes)
         end}
-    ] end}}.
+
+    ],
+    lists:foreach(fun({Name, Eval}) ->
+        lager:info("===== circle_and_spurs: ~s =====", [Name]),
+        Eval()
+    end, Tests).
 
 mixed_version_clusters() ->
-    case eunit:test(?MODULE:mixed_version_clusters_test_(), [verbose]) of
-        ok ->
-            pass;
-        error ->
-            % at the time this is written, the return value isn't acutally
-            % checked, the only way to fail is to crash the process.
-            % i leave the fail here in hopes a future version will actually
-            % do what the documentation says.
-            exit(error),
-            fail
+    case rt_config:config_or_os_env(run_rt_cascading_1_3_tests, false) of
```

Comment:
Are these tests still needed ?? Or do they need to be updated for "new" Riak versions like 2.0 -> 2.1?  Seems like these are specifically testing for 1.3 -> 1.4 changes.